dnsapi/dns_do.sh

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env sh
+
+# DNS API for Domain-Offensive / Resellerinterface / Domainrobot
+
```

Comment:
please add your github repository link here,  so that we can report issues to you once there is any bug found.